### PR TITLE
[scatter] more nice to haves

### PR DESCRIPTION
**Pull request by chejennifer**
- update the modal that pops up when user picks three variable. The new modal shows the user the third variable they picked and then forces them to keep one of the other two variables. if user closes the modal, no change.
- update the place options to look better on small screens

dev instance has been updated

### CODE REVIEW

File: static/js/tools/scatter2/place_options.tsx [hunk 1]
```diff
@@ -88,31 +88,31 @@ function PlaceOptions(): JSX.Element {
...
-                removePlace={() =>
-                  unselectEnclosingPlace(place, setChildPlaceTypes)
-                }
-                numPlacesLimit={1}
-              />
-            </div>
-          </Col>
-          <Col xs="auto">of type</Col>
-          <Col xs="3">
+        <div className="place-options-section place-search-section">
```
Comment (beets) [inline static/js/tools/scatter2/place_options.tsx:93]: feel free to use id's too - in this case, place-search-section and place-type-section are probably going to be unique identifiers

File: static/js/tools/scatter2/statvar.tsx [hunk 2]
```diff
@@ -106,62 +109,57 @@ function StatVarChooser(): JSX.Element {
...
-      <Modal
-        isOpen={!_.isEmpty(thirdStatVar.statVar)}
-        backdrop="static"
-        id="statvar-modal"
-      >
-        <ModalHeader close={null}>
-          Select two of the three statistical variables
-        </ModalHeader>
+      <Modal isOpen={modalOpen} backdrop="static" id="statvar-modal">
+        <ModalHeader toggle={closeModal}>Third Variable Selected</ModalHeader>
```
Comment (beets) [inline static/js/tools/scatter2/statvar.tsx:128]: Maybe a more actionable title? Some options:
- Please select two variables
- Only two variables supported

or a friendlier message

**Review by beets — APPROVED** [review]

File: static/js/tools/scatter2/place_options.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/tools/scatter2/place_options.tsx]: done

File: static/js/tools/scatter2/statvar.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/tools/scatter2/statvar.tsx]: done. I went with "Only two variables supported"

**Pull request merged**